nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/blob/ColdStorageHelper.java

```diff
@@ -71,6 +73,8 @@
 
     /**
      * Moves the main content associated with the document of the given {@link DocumentRef} to a cold storage.
+     * <p/>
+     * The permission {@value org.nuxeo.ecm.core.api.security.SecurityConstants#WRITE_COLD_STORAGE} is required.
      *
      * @return the updated document model if the move succeeds
      * @throws NuxeoException if the main content is already in the cold storage, or if there is no main content
```

Comment:
Can you please add the case where the current user doesn't have the required permission?

Reply:
Thanks! Nice catch! 🙂

Reply:
For debugging purposes I understand that it would be interesting to have more details.
But, if I am not mistaken, this exception will be sent to client side and, for security purposes, we should not provide too much details (IMHO).

Reply:
You are referring to https://github.com/nuxeo/nuxeo/pull/3843#discussion_r395772285.
Don't know if it's considered 
as a security issue or if we want to provide this information to the client, @efge WDYT?

Reply:
@ataillefer, not sure how the comment ended here 😅
Moved to the correct thread!

Reply:
It seems you haven't updated the javadoc for `@throws NuxeoException` to add the permission case.